### PR TITLE
Added Linter config; Added PropType checking; Added Travis-CI config

### PR DESCRIPTION
**Pull request by sutefan1**
I always use ESLint with Airbnb config as a base set of rules. The `eslintrc.json` gives an overview of excluded rules. In my opinion, those just make your life harder than it needs to be and thus they are deactivated. 
I added babel for the transpiling feature. Check out the `babelrc` for further information.
I use VSCode for coding and use the eslint extension, which makes sure that I see the linter errors inline. Big recommendation. 
I also added the script `npm run lint` for CI. It runs the local `eslint` on the whole project.

### CODE REVIEW

File: components/Credits.js [hunk 1]
```diff
@@ -1,64 +1,67 @@
...
   StyleSheet,
   Linking,
-  Image
-} from "react-native";
-import { Amplitude } from "expo";
-import { Analytics, Event } from "expo-analytics";
+  Image,
+} from 'react-native';
+import Proptypes from 'prop-types';
+import * as Expo from 'expo';
```
Comment (SKempin) [inline components/Credits.js:10]: Unsure on this change - please see comments

**Comment by SKempin** [comment]
Thanks so much! Just a couple of questions to clarify:

- Can we remove the trailing commas? 

- You have changed the import syntax of Amplitude, can you just explain why (I'm still learning!)? I've found [this article](https://forums.expo.io/t/import-expo-from-expo-has-been-deprecated/15799) on using that syntax.

**Comment by sutefan1** [comment]
> * Can we remove the trailing commas?

Updated the PR with the new linter rule. Please check out if this works for you now!
> * You have changed the import syntax of Amplitude, can you just explain why (I'm still learning!)? I've found [this article](https://forums.expo.io/t/import-expo-from-expo-has-been-deprecated/15799) on using that syntax.

As the article says there are two ways of importing stuff from expo:
- `import { XY } from 'expo';`
- `import * as Expo from 'expo';`
You have chosen to use the first one, which is no problem. The reason why I prefer the second one is for the sake of readability. You know at any point that `Amplitude` is an `Expo` function/object. The other reason is that you forgot sometimes to import `Expo` at all. This is often an issue if no linter or `strict mode` is in use.

In general there is no difference as there is no default export declared in `Expo` and you either destructure the exported object or give it a name with the `* as XY` syntax.

File: components/Credits.js (hunk 1 above)
Comment (sutefan1) [reply components/Credits.js:10]: Let me know if the comment below answers your question.

**Comment by SKempin** [comment]
Excellent stuff, thanks for your contribution and comments!

**Pull request merged**